travel-event-host-app/src/app/clients/user/auth-client.ts

```diff
@@ -0,0 +1,50 @@
+import { signIn } from 'next-auth/react';
+
+export async function clientSignUp(
```

Comment:
```suggestion
export async function signUpUser(
```

Reply:
no, the method is called signUp on Nextauth, it is a method on the client so clientSignUp